_data/changelogs/docs-maturity-model.yml

```diff
@@ -0,0 +1,16 @@
+title: Maturity model
+type: documentation
+changelogURL:
+items:
+  - date: 2022-03-10
+    summary: Replaced `uswds-gulp` with `uswds-compile`.
```

Comment:
```suggestion
    summary: Replaced the outdated text: `uswds-gulp` with updated: `uswds-compile`.
```

Reply:
Updated in: https://github.com/uswds/uswds-site/commit/afe4b145041a1e2f999bf84ce6cdcf5c6c39f208